.gitignore

```diff
@@ -1,9 +1,36 @@
+# Temporary files
+.tmp
+
 # Ignore some folders
+bower_components
 node_modules
 /static/framework
-/dist
+/static/build
+dist
+www
+
+# Logs
+logs
+*.log
+
+# Editors
+.idea
+.c9
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
```

Comment:
We don't have any code coverage tool for now, so let's remove the following lines